recipe/meta.yaml

```diff
@@ -1,24 +1,27 @@
-{% set version = "2.0.0" %}
+{% set version = "3.1.0" %}
+# Triton no longer tags releases.  We're taking the commit id from
+# PyTorch's .ci/docker/ci_commit_pins/triton.txt 
+{% set git_commit = "5fe38ffd73c2ac6ed6323b554205186696631c6f" %}
```

Comment:
What is the relation between commits in pytorch (per version tag) and triton versions? Are you proposing to build triton exclusively for pytorch, or is there some other source of that `3.1.0`?

https://github.com/pytorch/pytorch/blob/v2.5.1/.ci/docker/ci_commit_pins/triton.txt

Reply:
Unfortunately, upstream doesn't tag releases for a while now, and PyTorch sources happened to provide the best hint at which commit corresponds to the actual release. It's present on [the 3.1.x branch](https://github.com/triton-lang/triton/commit/5fe38ffd73c2ac6ed6323b554205186696631c6f), and `.py` files inside the wheel match the files in that commit. I suppose I could refer to the 3.1.x branch instead, because the commit is marked clearly enough there; I've left the link to PyTorch since that was the best hint.

Reply:
```suggestion
# Triton no longer tags releases, but there are release branches, e.g.
# https://github.com/triton-lang/triton/commits/release/3.1.x/
# Check if the commit id from Pytorch's latest pinned commit in
# https://github.com/pytorch/pytorch/blob/v{{ pytorch_ver }}/.ci/docker/ci_commit_pins/triton.txt
# can be found on one of those release branches, and use that as the version
{% set git_commit = "5fe38ffd73c2ac6ed6323b554205186696631c6f" %}
```